less/admin/AdminNav.less

```diff
@@ -1,17 +1,91 @@
-@admin-pane-width: 300px;
+@admin-pane-width: 250px;
 
 .App {
   padding-bottom: 0;
 }
-.AdminLinkButton-description {
-  display: none;
-}
+
 .AdminContent {
   padding: 20px 0;
 }
 .App-content .sideNavOffset {
   margin-top: 0;
 }
+
+.Header-controls {
+  > li {
+    margin-left: 10px;
+  }
+}
+
+.AdminNav-mainLink {
+  > .Button-label {
+    font-size: 14px !important;
```

Comment:
why `!important` here ? seems to only override a similar definition for desktop screens, we might as well change it there or remove it there and drop the `!important` here.